airflow/providers/google/cloud/transfers/bigquery_to_mysql.py

```diff
@@ -45,13 +48,15 @@ class BigQueryToMySqlOperator(BaseOperator):
         to MySQL
 
     **Example**: ::
-
+    
+       # [START howto_operator_bigquery_to_mysql]
        transfer_data = BigQueryToMySqlOperator(
             task_id='task_id',
             dataset_table='origin_bq_table',
             mysql_table='dest_table_name',
             replace=True,
         )
+        # [END howto_operator_bigquery_to_mysql]
```

Comment:
Use example in the docstring as BigQueryToMySqlOperator doesn't have its own system test, and creating one may not be necessary.